assets/src/edit-story/components/panels/design/preset/header.js

```diff
@@ -29,7 +29,7 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { Add, EditPencil } from '../../../../icons';
```

Comment:
Is the old pencil icon used anywhere else? If not, go ahead and delete it completely.

Reply:
Yep, it's used in the Document panel for media.